ui/src/app/styles/colors.ts

```diff
@@ -25,6 +25,7 @@ export default {
     dataSet: '#6CACE4',
     notebook: '#8BC990'
   },
+  border: '#b7b7b7',
```

Comment:
Have we checked with Lou before adding this to the color palette? In general, I think we should tend to use colorsWithWhiteness to avoid ending up with a large number of slight variations in `colors.tsx`. It will make it easier if we want to change the theming on the site to a different color palette.

Reply:
I did check with him. I can try and do the math on how to get `b7` out of `colorsWithWhiteness`.

Reply:
Yeah, I'm pushing for us to run a pretty tight ship on this file, because the refactor when we had 12 different blues was a pain in the neck, and I very much want to avoid doing that again.